core/packages/contracts/src/BeefyClient.sol

```diff
@@ -288,6 +288,41 @@ contract BeefyClient is Ownable {
         ticket.prevRandao = block.prevrandao;
     }
 
+    /**
```

Comment:
I preferred to have these functions where they were originally (at the end of the list of public functions), since they are really just helpers.